docs/pages/blog/date-pickers-stable-v5.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'The Date pickers gets a stable v5 release'
```

Comment:
```suggestion
title: 'The Date Pickers gets a stable v5 release'
```
@samuelsycamore, I'm a bit confused about the case on this one, since "Date Pickers" is not really a component but more like a reference to package(?).

Reply:
@joserodolfofreitas That's the last element to modify or not. Otherswise it's good for publiction